bin/lib/installable/archives.py

```diff
@@ -364,7 +364,7 @@ def __init__(self, install_context: InstallationContext, config: dict[str, Any])
         super().__init__(install_context, config)
 
     def fetch_and_pipe_to(self, staging: StagingDir, s3_path: str, command: list[str]) -> None:
-        untar_dir = staging.path
```

Comment:
this may have some knock-on effects for other installers....which might be just fine